src/main/groovy/org/shipkit/internal/gradle/ShipkitJavaPlugin.java

```diff
@@ -76,15 +76,17 @@ public void execute(Task t) {
             }
         });
 
-        //TODO we should have tasks from the same plugin to have the same group
-        //let's have a task maker instance in a plugin that has sets the group accordingly
-
-        final Task bintrayUploadAll = TaskMaker.task(project, "bintrayUploadAll", new Action<Task>() {
-            public void execute(Task t) {
-                t.setDescription("Depends on all 'bintrayUpload' tasks from all Gradle projects.");
-                //It is safer to run bintray upload after git push (hard to reverse operation)
-                //This way, when git push fails we don't publish jars to bintray
-                t.shouldRunAfter(GitPlugin.GIT_PUSH_TASK);
+        final Task performRelease = TaskMaker.task(project, "performRelease", new Action<Task>() {
+            public void execute(final Task t) {
+                t.setDescription("Performs release. " +
+                        "Ship with: './gradlew performRelease -Pshipkit.dryRun=false'. " +
+                        "Test with: './gradlew testRelease'");
```

Comment:
"testRelease" additionally schedules clean up tasks. dryRun by default was made to avoid accidental publications. However, it is very counter-intuitive and I think we should improve it (I have some ideas).